### DISCUSSION TITLE
Full API overview

### DISCUSSION THREAD

**Discussion by omarcostahamido** [Qiskit/.github#19] [Documentation]
(Sorry if I am late to the game)
I wrote [here](https://qiskit.slack.com/archives/C7SJ0PJ5A/p1645290818003019?thread_ts=1637222710.343500&cid=C7SJ0PJ5A) expressing my need for a _single API_ that can list everything there is in qiskit, showing each component, and sub-components, in an expandable tree directory. Each item on it would be linked to the appropriate documentation page, of course.

**Comment by 1ucian0** [comment]
The API documentation is generated with Sphinx. So you could generate a _table of content_ locally:
```
.. contents:: Table of Contents
    :depth: 3
```

Check out how to build the documentation locally [here](https://qiskit.github.io/qiskit_sphinx_theme/apidocs/index.html).

**Reply by omarcostahamido** [reply]
thank you.
